data/crac-creation/crac-creator-cse/src/test/java/com/farao_community/farao/data/crac_creation/creator/cse/CseCracCreatorWithMneTest.java

```diff
@@ -0,0 +1,204 @@
+        assertNull(cseCriticalBranchCreationContext.getCreatedCnecsIds().get(Instant.CURATIVE));
+        assertTrue(cseCriticalBranchCreationContext.isBaseCase());
+        assertTrue(cseCriticalBranchCreationContext.getContingencyId().isEmpty());
+        assertEquals(fromNode, cseCriticalBranchCreationContext.getNativeBranch().getFrom());
+        assertEquals(toNode, cseCriticalBranchCreationContext.getNativeBranch().getTo());
+        assertEquals(suffix, cseCriticalBranchCreationContext.getNativeBranch().getSuffix());
+        assertEquals(cnecPreventiveId, cseCriticalBranchCreationContext.getCreatedCnecsIds().get(Instant.PREVENTIVE));
+        assertEquals(inverted, cseCriticalBranchCreationContext.isDirectionInvertedInNetwork());
+    }
+
+    public void assertMneWithContingencyInCrac(String name, String fromNode, String toNode, String suffix, String direction, String contingencyId, Instant instant, double expectedIMax, double expectedThreshold, Unit expectedThresholdUnit, double expectedNV) {
+        String createdCnecId = String.format("%s - %s ->%s   - %s - %s", name, fromNode, toNode, contingencyId, instant);
+        String nativeId = String.format("%s - %s  - %s  - %s", name, fromNode, toNode, contingencyId);
+        Crac crac = cracCreationContext.getCrac();
+        assertNotNull(crac.getFlowCnec(createdCnecId));
+        assertEquals(name, crac.getFlowCnec(createdCnecId).getName());
+        assertTrue(crac.getFlowCnec(createdCnecId).isMonitored());
+        assertFalse(crac.getFlowCnec(createdCnecId).isOptimized());
+        assertEquals(expectedIMax, crac.getFlowCnec(createdCnecId).getIMax(Side.LEFT), 0.00001);
+        assertEquals(expectedIMax, crac.getFlowCnec(createdCnecId).getIMax(Side.RIGHT), 0.00001);
+        FlowCnec flowCnec = crac.getFlowCnec(createdCnecId);
+        assertTrue(hasThreshold(nativeId, expectedThreshold, expectedThresholdUnit, flowCnec, direction, Side.LEFT));
+        assertTrue(hasThreshold(nativeId, expectedThreshold, expectedThresholdUnit, flowCnec, direction, Side.RIGHT));
+        assertEquals(crac.getFlowCnec(createdCnecId).getState().getInstant(), instant);
+        assertEquals(crac.getFlowCnec(createdCnecId).getNetworkElement().getId(), crac.getFlowCnec(createdCnecId).getNetworkElement().getName());
+        boolean directionInvertedInNetwork = cracCreationContext.getBranchCnecCreationContext(nativeId).isDirectionInvertedInNetwork();
+        if (directionInvertedInNetwork) {
+            assertEquals(toNode + "  " + fromNode + "  " + suffix, crac.getFlowCnec(createdCnecId).getNetworkElement().getId());
+        } else {
+            assertEquals(fromNode + "  " + toNode + "  " + suffix, crac.getFlowCnec(createdCnecId).getNetworkElement().getId());
+
+        }
+        assertEquals(crac.getFlowCnec(createdCnecId).getNominalVoltage(Side.RIGHT), crac.getFlowCnec(createdCnecId).getNominalVoltage(Side.RIGHT));
```

Comment:
I guess you intended to compare LEFT & RIGHT here ? 
either way, better compare both left & right to expectedNV, for a better readability of the assertions